backend/templates/index.html

```diff
@@ -201,7 +201,11 @@ <h4 style="margin-top: 0px;">QLever UI Shortcuts:</h4>
                     <i class="glyphicon glyphicon-align-left"></i> Examples
                     <span class="caret"></span>
                   </button>
-                  <ul class="dropdown-menu">
+                  <ul id="example-list" class="dropdown-menu">
+                    <div style="padding: 10px 20px;">
+                      <input id="example-keyword-search" oninput="filterExamplesDebounced(event)" type="text" placeholder="search for keywords" style="width: 100%"> </input>
```

Comment:
Would be great to have the focus automatically set on this whenever the Example button is pressed

Reply:
@hannahbast thanks for the suggestion, i implemented this and it feels very smooth and convenient.